suite-native/module-accounts/src/components/AccountListItem.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { TouchableOpacity } from 'react-native';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { AccountsRootState, selectAccountName } from '@suite-common/wallet-core';
+import { Box, Text } from '@suite-native/atoms';
+import {
+    AccountsStackParamList,
+    AccountsStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { Account } from '@suite-common/wallet-types';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { CryptoIcon } from '@trezor/icons';
+
+type AccountListItemProps = {
+    account: Account;
+};
+
+const accountListItemStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.gray0,
+    padding: utils.spacings.medium,
+    borderRadius: 12,
+    marginBottom: utils.spacings.small,
+}));
+
+export const AccountListItem = ({ account }: AccountListItemProps) => {
+    const navigation =
+        useNavigation<StackNavigationProps<AccountsStackParamList, AccountsStackRoutes.Accounts>>();
+    const { applyStyle } = useNativeStyles();
+    const accountName = useSelector((state: AccountsRootState) =>
+        // eslint-disable-next-line no-restricted-syntax
+        selectAccountName(state, account.key),
+    );
+
+    const handleSelectAccount = (key: string) => {
+        navigation.navigate(AccountsStackRoutes.AccountDetail, {
+            accountKey: key,
+        });
+    };
+
+    return (
+        <TouchableOpacity onPress={() => handleSelectAccount(account.key)}>
+            <Box
+                flexDirection="row"
+                justifyContent="space-between"
+                alignItems="center"
+                style={applyStyle(accountListItemStyle)}
+            >
+                <Box flexDirection="row">
+                    <Box marginRight="small">
+                        <CryptoIcon name="btc" />
+                    </Box>
+                    <Text color="gray800">{accountName}</Text>
+                </Box>
+                <Box alignItems="flex-end">
+                    <Text color="gray800" variant="hint">
+                        $ {account.formattedBalance}
```

Comment:
Might be better to use template literals for better grouping. The dollar sign in this case looks a bit random in my opinion.

```suggestion
                        {`$ ${account.formattedBalance}`}
```

Reply:
Perhaps it's a patters worth adopting in general?

Reply:
I think this comes down to personal choice. IMO without the template literal it looks much cleaner and as a developer it's easier and faster for me to see the expected result. I can change it if you want, but you are right, should be adopted as general pattern in code style, if we decide to go this way.

Reply:
This will be replaced with formatter very soon anyway.